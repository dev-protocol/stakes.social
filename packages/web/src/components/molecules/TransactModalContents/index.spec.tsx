import React from 'react'
import { render } from '@testing-library/react'
import { TransactModalContents } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-for-apps/hooks')
jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/wallet/hooks')
jest.mock('ethers')
jest.mock('web3')
jest.mock('@dev/graphql')

describe(`${TransactModalContents.name}`, () => {
  test('Snapshot', () => {
    const component = render(<TransactModalContents type="stake" propertyAddress="0x0" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
