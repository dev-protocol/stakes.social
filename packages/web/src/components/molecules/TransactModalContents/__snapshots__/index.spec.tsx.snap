// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactModalContents Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-dJjYzT dgosBC"
    >
      <div
        class="sc-hGPBjI ftvIut"
      >
        <div
          class="sc-bqiRlB efrysm"
        >
          <div
            class="sc-ksdxgE fRLOSX"
            size="60"
          />
        </div>
        <h4
          class="sc-bkkeKt sc-kfPuZi hjVgaQ fqgYZg"
        >
          0x0-name
        </h4>
        <span
          class="sc-fKVqWL khTKhP"
        >
          0x0
        </span>
      </div>
      <div
        class="sc-bdvvtL frQklk"
      >
        <div
          class="sc-gKclnd fAyBB"
          style="opacity: 0.3;"
        >
          <div
            class="ant-radio-group ant-radio-group-outline"
            style="margin-bottom: 12px;"
          >
            <div
              class="ant-space ant-space-vertical"
            >
              <div
                class="ant-space-item"
                style="margin-bottom: 8px;"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      type="radio"
                      value="-1"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    <span
                      style="margin-left: 12px;"
                    >
                      New position
                    </span>
                  </span>
                </label>
              </div>
              <div
                class="ant-space-item"
                style="margin-bottom: 8px;"
              >
                )
              </div>
              <div
                class="ant-space-item"
                style="margin-bottom: 8px;"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      type="radio"
                      value="2"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    <div
                      class="ant-space ant-space-horizontal ant-space-align-center"
                    >
                      <div
                        class="ant-space-item"
                        style="margin-right: 24px;"
                      >
                        <div
                          style="margin-left: 8px; min-width: 40px;"
                        >
                          #
                          2
                        </div>
                      </div>
                      <div
                        class="ant-space-item"
                        style="margin-right: 24px;"
                      >
                        <div
                          style="min-width: 120px;"
                        >
                          <span>
                            0.000000000000001
                          </span>
                          <span
                            style="font-size: 0.6em; margin-left: 2px;"
                          >
                            Staked
                          </span>
                        </div>
                      </div>
                      <div
                        class="ant-space-item"
                      >
                        <div>
                          0
                          <span
                            style="font-size: 0.6em; margin-left: 2px;"
                          >
                            Claimable
                          </span>
                        </div>
                      </div>
                    </div>
                  </span>
                </label>
              </div>
              <div
                class="ant-space-item"
                style="margin-bottom: 8px;"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      type="radio"
                      value="4"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    <div
                      class="ant-space ant-space-horizontal ant-space-align-center"
                    >
                      <div
                        class="ant-space-item"
                        style="margin-right: 24px;"
                      >
                        <div
                          style="margin-left: 8px; min-width: 40px;"
                        >
                          #
                          4
                        </div>
                      </div>
                      <div
                        class="ant-space-item"
                        style="margin-right: 24px;"
                      >
                        <div
                          style="min-width: 120px;"
                        >
                          <span>
                            0.000000000000001
                          </span>
                          <span
                            style="font-size: 0.6em; margin-left: 2px;"
                          >
                            Staked
                          </span>
                        </div>
                      </div>
                      <div
                        class="ant-space-item"
                      >
                        <div>
                          0
                          <span
                            style="font-size: 0.6em; margin-left: 2px;"
                          >
                            Claimable
                          </span>
                        </div>
                      </div>
                    </div>
                  </span>
                </label>
              </div>
              <div
                class="ant-space-item"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      type="radio"
                      value="6"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    <div
                      class="ant-space ant-space-horizontal ant-space-align-center"
                    >
                      <div
                        class="ant-space-item"
                        style="margin-right: 24px;"
                      >
                        <div
                          style="margin-left: 8px; min-width: 40px;"
                        >
                          #
                          6
                        </div>
                      </div>
                      <div
                        class="ant-space-item"
                        style="margin-right: 24px;"
                      >
                        <div
                          style="min-width: 120px;"
                        >
                          <span>
                            0.000000000000001
                          </span>
                          <span
                            style="font-size: 0.6em; margin-left: 2px;"
                          >
                            Staked
                          </span>
                        </div>
                      </div>
                      <div
                        class="ant-space-item"
                      >
                        <div>
                          0
                          <span
                            style="font-size: 0.6em; margin-left: 2px;"
                          >
                            Claimable
                          </span>
                        </div>
                      </div>
                    </div>
                  </span>
                </label>
              </div>
            </div>
          </div>
          <span
            class="ant-input-affix-wrapper ant-input-affix-wrapper-lg sc-dkPtRN kCszbf"
          >
            <input
              class="ant-input ant-input-lg"
              id="stake"
              type="number"
              value=""
            />
            <span
              class="ant-input-suffix"
            >
              <button
                class="sc-gsDKAQ kExfsU"
              >
                DEV
              </button>
            </span>
          </span>
          <div
            class="sc-eCImPb UUa-dQ"
          >
            <div
              class="ant-row"
            >
              <div
                class="ant-col ant-col-11"
              >
                <button
                  class="ant-btn ant-btn-primary sc-hKwDye jTvuMH"
                  disabled=""
                  type="button"
                >
                  <span>
                    Approve
                  </span>
                </button>
              </div>
              <div
                class="ant-col ant-col-2 sc-jRQBWg gyNKvI"
              >
                â†’
              </div>
              <div
                class="ant-col ant-col-11"
              >
                <button
                  class="ant-btn ant-btn-primary sc-hKwDye jTvuMH"
                  disabled=""
                  type="button"
                >
                  <span>
                    Stake
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          style="height: 40px;"
        />
      </div>
    </div>
  </div>
</body>
`;
